modules/oe_theme_content_person/src/Plugin/ExtraField/Display/PersonJobLabelExtraField.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_person\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+
+/**
+ * Display Person job label.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_theme_content_person_job_label",
+ *   label = @Translation("Person job label"),
+ *   bundles = {
+ *     "oe_person_job.oe_default",
+ *   },
+ *   visible = true
+ * )
+ */
+class PersonJobLabelExtraField extends ExtraFieldDisplayFormattedBase {
```

Comment:
As discussed with @yenyasinn we will try to remove this in favor of the entity revision reference label formatter.

Reply:
Label is shown in the Person Job full view mode itself. So we show it as an extra field.

<img width="1324" alt="Screenshot 2021-03-26 at 13 00 36" src="https://user-images.githubusercontent.com/1183951/112615445-8c1b2e80-8e33-11eb-8da2-c17161a21311.png">

Reply:
Got it, thanks!